src/common/camera.h

```diff
@@ -16,12 +16,17 @@
 
 class camera {
   public:
-    void initialize(double aspect_ratio = 1.0) {
+    void initialize(double image_width, double image_height) {
+        auto aspect_ratio = image_width / image_height;
         auto theta = degrees_to_radians(vfov);
         auto h = tan(theta/2);
+
         auto viewport_height = 2.0 * h;
         auto viewport_width = aspect_ratio * viewport_height;
 
+        pixel_delta_x = 1.0 / (image_width-1);
+        pixel_delta_y = 1.0 / (image_height-1);
```

Comment:
shouldn't this just be 1.0 / image_height ?

In the case where there are two pixels, shouldn't delta be 1/2?

Reply:
See distinct comment in the main PR chain

Reply:
Here, the pixel delta is the stride from one pixel location to the next. The intent is to have a delta that can be multiplied by the pixel index, and added to the image origin to get the location of the indexed pixel.